src/main/java/net/troja/eve/esi/HeaderUtil.java

```diff
@@ -0,0 +1,82 @@
+package net.troja.eve.esi;
+
+import com.fasterxml.jackson.databind.util.ISO8601DateFormat;
+import java.text.ParseException;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+public class HeaderUtil {
+
+    private static final ISO8601DateFormat DATE_FORMAT = new ISO8601DateFormat();
+
+    private HeaderUtil() { }
+
+    /**
+     * 
+     * @param responseHeaders ApiClient.getResponseHeaders()
+     * @return 
+     */
+    public static Integer getXPages(Map<String, List<String>> responseHeaders) {
+        //Get header
+        String header = getHeader(responseHeaders, "X-Pages");
```

Comment:
If we want to keep this code long term, suggest you make "X-Pages" and "Expires" into final static constants.

Reply:
Good suggestion, will update the PR